insights/client/utilities.py

```diff
@@ -235,7 +242,10 @@ def magic_plan_b(filename):
 
 def generate_container_id(container_name):
     # container id is a uuid in the namespace of the machine
-    return str(uuid.uuid5(uuid.UUID(generate_machine_id()), container_name.encode('utf8')))
```

Comment:
It used to be that container IDs were based on their host machine, so if we don't need to see the Container IDs then the call to "generate_machine_id()" can be removed. So either we need to re-think the "generate_machine_id()" functionality or just remove the call to that function here. Note that "generate_machine_id()" did not always return "None" when "container_mode" or "analyze_image" was True.